rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/MysqlTableMetaCache.java

```diff
@@ -102,6 +102,7 @@ protected TableMeta fetchSchema(DataSource dataSource, String tableName) throws
 
     private TableMeta resultSetMetaToSchema(ResultSetMetaData rsmd, DatabaseMetaData dbmd, String tableName)
         throws SQLException {
+        //always "" for mysql
```

Comment:
What does this sentence mean?

Reply:
That means this method always return "" in Mysql. Mysql does not implement this interface.